app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[0] != "mv" or len(parts) != 3:
+        raise ValueError("Command format is not correct")
+
+    source_file, path = parts[1], parts[2]
+
+    if not os.path.exists(source_file):
+        raise FileNotFoundError(f"No such file or directory: {source_file}")
```

Comment:
The `FileNotFoundError` is raised if the source file does not exist, which is appropriate. Ensure that the path provided is correct and accessible.